core/src/main/java/org/apache/carbondata/core/keygenerator/columnar/impl/MultiDimKeyVarLengthEquiSplitGenerator.java

```diff
@@ -143,17 +142,18 @@ private void intialize() {
 
   private int[] convertToArray(List<Integer> list) {
```

Comment:
After second thought, I think these kind of functions are unnecessary, we should use
```
    List<Integer> list = new LinkedList<>();
    Integer[] array = x.toArray(new Integer[list.size()]);
```
from JDK instead of writing our own code